[Wrangler] Document how `.dev.vars` works in Wrangler (#5123)
cloudflare/cloudflare-docs: content/workers/wrangler/configuration.md

If your IT team has configured your computer's proxy settings, be aware that the first non-empty environment variable in this list will be used when Wrangler makes outgoing requests.

For example, if both `https_proxy` and `http_proxy` are set, Wrangler will only use `https_proxy` for outgoing requests.

## Local environments

Wrangler allows you to set variable or secret keys locally without the need to set the `[vars]` bindings in your `wrangler.toml` file. The variables and secrets keys you set only get used when you run `wrangler dev`.

> You need to create a file called `.dev.vars` next to the `wrangler.toml` file (or in the current working directory if there is no `wrangler.toml`). 

Reviewer: ```suggestion
You need to create a file called `.dev.vars` next to the `wrangler.toml` file (or in the current working directory if there is no `wrangler.toml`. 
```

If your IT team has configured your computer's proxy settings, be aware that the first non-empty environment variable in this list will be used when Wrangler makes outgoing requests.

For example, if both `https_proxy` and `http_proxy` are set, Wrangler will only use `https_proxy` for outgoing requests.

## Local Environments

> Wrangler allows you to set variable or secret keys locally. They only get used when you run `wrangler dev` and it is useful during development without the need to set the `[vars]` bindings in your `wrangler.toml` file.

Reviewer: ```suggestion
Wrangler allows you to set variable or secret keys locally without the need to set the `[vars]` bindings in your `wrangler.toml` file. The variables and secrets keys you set only get used when you run `wrangler dev`.
```
